### PR TITLE
Convert library landing page update

### PR DESCRIPTION
**Pull request by jpsiitonen**

### CODE REVIEW

File: sdk/lib/convert/convert.dart [hunk 1]
```diff
@@ -17,15 +17,79 @@
...
+///
+/// ## UTF-8
+/// A [Utf8Codec] encodes strings to utf-8 code units (bytes) and decodes
+/// UTF-8 code units to strings.
+///
+/// The [utf8] encoder/decoder transforms between Strings and bytes.
+///
+/// Example:
+/// ```dart
+/// var encoded = utf8.encode("Îñţérñåţîöñåļîžåţîờñ");
```
Comment (alexlindroos) [inline sdk/lib/convert/convert.dart]: you can use single quotes

File: sdk/lib/convert/convert.dart [hunk 2]
```diff
@@ -17,15 +17,79 @@
...
+///
+/// The [json] encoder/decoder transforms between strings and
 /// object structures, such as lists and maps, using the JSON format.
 ///
-/// UTF-8 is a common variable-width encoding that can represent
-/// every character in the Unicode character set.
-/// The UTF-8 encoder/decoder transforms between Strings and bytes.
+/// Examples
+/// ```dart
+/// var encoded = json.encode([1, 2, { "a": null }]);
```
Comment (alexlindroos) [inline sdk/lib/convert/convert.dart:32]: Should we put the output as comment or print in all examples?

File: sdk/lib/convert/convert.dart [hunk 3]
```diff
@@ -17,15 +17,79 @@
...
+/// ```
+/// For more information see also [Utf8Encoder] and [Utf8Decoder].
+///
+/// ## ASCII
+/// An [AsciiCodec] allows encoding strings as ASCII bytes
+/// and decoding ASCII bytes to strings.
+///
+/// Example:
+/// ```dart
+/// var encoded = ascii.encode("This is ASCII!");
```
Comment (alexlindroos) [inline sdk/lib/convert/convert.dart]: Single quotes

File: sdk/lib/convert/convert.dart [hunk 4]
```diff
@@ -17,15 +17,79 @@
...
+/// decodes using both the base64 and base64url alphabets,
+/// does not allow invalid characters and requires padding.
+///
+/// The [base64] encoder/decoder transforms between strings and Base64 bytes.
+///
+/// Example:
+/// ```dart
+/// var encoded = base64.encode([0x62, 0x6c, 0xc3, 0xa5, 0x62, 0xc3, 0xa6,
+///                              0x72, 0x67, 0x72, 0xc3, 0xb8, 0x64]);
+/// var decoded = base64.decode("YmzDpWLDpnJncsO4ZAo=");
```
Comment (alexlindroos) [inline sdk/lib/convert/convert.dart]: single quotes

File: sdk/lib/convert/convert.dart [hunk 5]
```diff
@@ -17,15 +17,79 @@
...
+///
+/// ## Latin-1
+/// A [Latin1Codec] encodes strings to ISO Latin-1 (aka ISO-8859-1) bytes
+/// and decodes Latin-1 bytes to strings.
+///
+/// The [latin1] encoder/decoder transforms between strings and Latin-1 bytes.
 ///
+/// Example:
+/// ```dart
+/// var encoded = latin1.encode("blåbærgrød");
```
Comment (alexlindroos) [inline sdk/lib/convert/convert.dart]: single quotes

File: sdk/lib/convert/convert.dart (hunk 2 above)
Comment (jpsiitonen) [reply sdk/lib/convert/convert.dart:32]: probably not needed here.

**Pull request closed without merging**